.github/workflows/copyright-test.yml

```diff
@@ -0,0 +1,35 @@
+name: Copyright
```

Comment:
Can we get dots in front of these filenames?

Reply:
I guess there's no reason not, but they're in a `.github/workflows` directory, and not sure if we gain anything by having hidden files in a non-hidden directory, in a hidden directory!

Reply:
Good point! Hadn't seen that